bump/action.yml

```diff
@@ -45,6 +45,12 @@ inputs:
     description: 'Optional SemVer prefix (eg. "", "v", "componentX-"). If set, only tags with this exact prefix shall be considered. "*" is a special value, meaning the closest SemVer is used, regardless of prefix".'
     default: '*'
     required: false
+  sdkver-release-type:
```

Comment:
Wouldn't it make sense to call it more generically `pre-release-type` (or `prerelease-identifier` or `prerelease-suffix`)? Then it would be more agnostic of the selected version-scheme, as it could equally be applicable for semver in the end if we would like to support there pre-releases?

Reply:
> Wouldn't it make sense to call it more generically `pre-release-type` (or `prerelease-identifier` or `prerelease-suffix`)?

`pre-release`-anything isn't a great name for an input that also allows you to create a full release.
SdkVer defines three "release" types, hence this name.
A more generic `bump-type` or `release-type` is maybe _too_ generic..

> as it could equally be applicable for semver in the end if we would like to support there pre-releases

Problem is the "could/would" part. To my mind, it would only make sense to call it that if there are concrete plans to incorporate support for it in SemVer.
The reason I chose this SdkVer-specific name is because a more generic name might lead people to expect it to also work for SemVer. Although that can be solved in the description.

Reply:
I would personally like `bump-type` or `release-type` more than the strongly schema typed variant, as I think it would make sense to handle the specifics if the input is relevant for a schema to the code handling the inputs. In that way you won't need to think about deprecation of the input parameter whenever we would like to start supporting SemVer schema also in future.

Likely even better would be if we could avoid this input parameter at all as an user, but that would be a bridge too far for the initial implementation I suppose. I know that @KevinDeJong-TomTom would like to see this also ;)